src/NUnitTestAdapter/Internal/Extensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace NUnit.VisualStudio.TestAdapter.Internal
+{
+#if NET35
+
```

Comment:
Nitpick: empty line between the `#if` and the class, but no empty line between the `#endif` and either class.

Reply:
removed empty line